src/LSCargo.Web/Views/Cms/Contact.cshtml

```diff
@@ -0,0 +1,38 @@
+﻿@model LSCargo.Web.Models.ContactPage
+@inject Piranha.IApi Api
+@{
+    ViewBag.Title = Model.Title;
+    ViewBag.HasImage = Model.Heading.PrimaryImage.HasValue;
+
+    string PrimaryImage()
+    {
+        if (Model.Heading.PrimaryImage.HasValue)
+        {
+            return $"style=background-image:url({Url.Content(Model.Heading.PrimaryImage)})";
+        }
+        return "";
+    }
+}
+
+<section class="jumbotron jumbotron-fluid" @PrimaryImage()>
+    <div class="container">
+        <div class="row">
+            <div class="text-light col-sm-9">
+                <h1 class="display-4">@Model.Title</h1>
+                <p class="lead">
+                    @Html.Raw(Model.Heading.Ingress)
+                </p>
+            </div>
+        </div>
+    </div>
+</section>
+
+<div class="main">
+    <div class="container">
+        @Html.DisplayFor(m => m.Blocks)
+
+        <div class="row">
```

Comment:
Här är markup fel. Se Views/Prototype/Contact.cshtml för korrekt markup. Borde vara nånting i stil med:

    <div class="row justify-content-center">
        <div class="col-lg-9">
            <ul class="contact-list row justify-content-center">
                @foreach (var contactCard in Model.ContactCards)
                {
                    <li class="contact-list__item col-md-4">
                        @Html.DisplayFor(m => contactCard)
                    </li>
                }
            </ul>
        </div>
    </div>